x-pack/plugins/cross_cluster_replication/public/app/components/remote_clusters_form_field.js

```diff
@@ -143,6 +143,12 @@ export class RemoteClustersFormField extends PureComponent {
           hasNoInitialSelection={!hasClusters}
           isInvalid={areErrorsVisible && Boolean(errorMessage)}
           data-test-subj="remoteClusterSelect"
+          aria-label={i18n.translate(
+            'xpack.crossClusterReplication.remoteClustersFormField.remoteClusterSelectAriaLabel',
+            {
+              defaultMessage: 'Remote cluster',
```

Comment:
What do you think about making this a little more descriptive? Something like "Remote cluster selection" or "Remote cluster dropdown"?

Reply:
Good point! I think "Remote cluster selection" is a suitable description. Added with [5f0f829](https://github.com/elastic/kibana/pull/169511/commits/5f0f8294d896ddf3565b4db3d2a78f701d122857)